examples/dbc2val/dbcreader.py

```diff
@@ -62,9 +62,9 @@ def rxWorker(self):
             try:
                 decode=self.db.decode_message(msg.arbitration_id, msg.data)
                 #print("Decod" +str(decode))
-            except:
+            except Exception as e:
                 self.parseErr+=1
-                #print("Error Decoding")
+                #print("Error Decoding: "+str(e))
```

Comment:
why commented out and changed?

Reply:
It was also commented out before. It is really only usable,for debugging, otherwise you get spammed with hundreds of msgs/seconds (IDs not defined in DBC are also one kind of decoding error). The change just gives more information about the kind of error, when commented in